python/lsst/ts/atdome/dome_csc.py

```diff
@@ -566,34 +545,26 @@ async def connect(self):
                     port = self.mock_ctrl.port
                 else:
                     port = self.config.port
-                connect_coro = asyncio.open_connection(host=host, port=port)
-                self.reader, self.writer = await asyncio.wait_for(
-                    connect_coro, timeout=self.config.connection_timeout
+                self.client = tcpip.Client(host=host, port=port, log=self.log)
+                print("Constructed the client; waiting to connect")
+                await asyncio.wait_for(
+                    self.client.start_task, timeout=self.config.connection_timeout
                 )
+                print("Connected; wait for prompt")
                 # drop welcome message
                 await asyncio.wait_for(
-                    self.reader.readuntil(">".encode()),
-                    timeout=self.config.read_timeout,
+                    self.client.readuntil(b">"), timeout=self.config.read_timeout
                 )
+            print("Fully connected")
```

Comment:
Is this print necessary? There is a debug log statement in the next line. Maybe change that to info instead if you really want output?

Reply:
Oops. Not necessary -- it was debugging code. I'll take it out and look for other print statements.